tasks/views.py

```diff
@@ -15,8 +15,24 @@ def view_tasks(request):
 
     if request.method == "POST":
         priority = request.POST.get('priority')
+        status = request.POST.get('status')
+
+        if status == 'InProgress':
+            status = 'in_progress'
```

Comment:
If you have some spare time consider to check out my suggestion above instead of this if statement, I think it will work.